clang/test/AST/ByteCode/libcxx/memcmp-pointer.cpp

```diff
@@ -0,0 +1,120 @@
+// RUN: %clang_cc1 -std=c++2c -fexperimental-new-constant-interpreter -verify=expected,both %s
+// RUN: %clang_cc1 -std=c++2c  -verify=ref,both %s
+
+// both-no-diagnostics
+
+namespace std {
+inline namespace {
```

Comment:
What is this funny construct?

Reply:
Just a reduced test case. Anything special about this line?

Reply:
I'm just wondering what the meaning of an inline anonymous namespace is.

Reply:
Oh, no meaning at all, I think cvise even removed the `inline` and I added it back so the compiler wouldn't complain.